runtime/calamari/src/lib.rs

```diff
@@ -916,8 +939,36 @@ pub type Executive = frame_executive::Executive<
 	frame_system::ChainContext<Runtime>,
 	Runtime,
 	AllPalletsReversedWithSystemFirst,
+	SchedulerMigrationV3,
 >;
 
+// Migration for scheduler pallet to move from a plain Call to a CallOrHash.
+pub struct SchedulerMigrationV3;
+
+impl OnRuntimeUpgrade for SchedulerMigrationV3 {
+	fn on_runtime_upgrade() -> frame_support::weights::Weight {
+		Scheduler::migrate_v1_to_v3()
```

Comment:
I just checked the `scheduler.storageVersion` on calamari mainnet, it's `V1`. But kusama and polkadot are `V2`.
Somehow, we didn't notice this migration from `V1` to `V2`.

Reply:
Yes, that's why I'm using `migrate_v1_to_v3()` here.

Reply:
I'm going to add checking for migrations to our checklists, because we shouldn't miss these.